rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/explorer/UnityExplorerNode.kt

```diff
@@ -123,17 +145,18 @@ open class UnityExplorerNode(project: Project,
         return true
     }
 
-    class Root(project: Project, virtualFile: VirtualFile)
+    class AssetsRoot(project: Project, virtualFile: VirtualFile)
         : UnityExplorerNode(project, virtualFile, listOf()) {
 
         private val referenceRoot = ReferenceRoot(project)
 
         override fun update(presentation: PresentationData) {
             if (!virtualFile.isValid) return
-            presentation.presentableText = project!!.solutionName
+            presentation.presentableText = "Assets"
```

Comment:
Why 'Assets' instead of solution name?

Reply:
Because it describes what's there - this node shows Assets, the other node shows Packages. It was a bit weird having "GemShader" and "Packages". And the project name is available in the title bar, etc.

Reply:
But if there is no 'Packages' folder, the root node 'Assets' also looks strange... at least for me